crates/holochain_state/src/scratch.rs

```diff
@@ -47,11 +49,23 @@ impl Scratch {
         Self::default()
     }
 
-    pub fn add_header(&mut self, item: SignedHeaderHashed) {
+    pub fn chain_top_ordering(&self) -> ChainTopOrdering {
+        self.chain_top_ordering
+    }
+
+    pub fn respect_chain_top_ordering(&mut self, chain_top_ordering: ChainTopOrdering) {
```

Comment:
I don't really get what this is doing? The default for `ChainTopOrdering` is `Strict` so when a new scratch is created this is already strict and this function doesn't do anything unless it's `Strict`.
Is the plan to add another constructor to create a `Relaxed` scratch in a follow up PR?